src/components/Box/Box.module.scss

```diff
@@ -1,13 +1,19 @@
 // Monday ui styles dependencies
 @import "~monday-ui-style/src/common";
+@import "~monday-ui-style/dist/mixins";
 
 // Utilities map
 @import "../../styles/utilities/utilities-settings";
 
-.box{
+.box {
   overflow: hidden;
+
+  &.scrollable {
+    /* TODO: make default in the next major version */
+    overflow: auto;
```

Comment:
Oh we are adding this? Why aren't we letting people to do it on their own?

Reply:
See line 9 :)